src/codeu/chat/common/ServerInfoTest.java

```diff
@@ -0,0 +1,22 @@
+package codeu.chat.common;
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+import codeu.chat.util.Uuid;
+
+import java.io.IOException;
+
+public class ServerInfoTest {
+
+    @Test
+    public void testServerInfo() {
+        ServerInfo server1 = new ServerInfo();
```

Comment:
It's awesome to see a test class! Nice work!

The only thing I'm worried about is now anytime anybody changes the server version, they'll also have to change the test class. Maybe the `ServerInfo` class should always take the version as a parameter? What do you think?